specs/clustertasks/clustertask-s2i.spec

```diff
@@ -6,7 +6,7 @@ Pre condition:
 
 
 ## S2I nodejs pipelinerun: PIPELINES-14-TC01
-Tags: e2e, clustertasks, non-admin, s2i
+Tags: e2e, clustertasks, non-admin, s2i, sanity, interop
```

Comment:
what is the relationship between `sanity` and `interop` - can't we use `sanity` for interop testing?

Reply:
Even I thought about this but later decided to have separate tags for each. I do not have strong opinion on having different tags for sanity and interop :)